CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ All notable changes to this project will be documented in this file. The format
 - Key results can now be rearranged by drag and drop.
 - New key results are now given a start value of 0, a target value of 100, and
   percentage as unit of measurement by default.
+- Added filter for ownerOptions on keyResults
```

Comment:
Tenker at denne kan virke ganske kryptisk for sluttbruker. :stuck_out_tongue: Tenker at vi kanskje heller kan skrive en lenger tekst for hele featuren med bidragsytere osv. når feature branchen er klar.